head/firmware/main.cpp

```diff
@@ -165,7 +169,9 @@ adc::ADC_interface ADC_intf1 = {
 
 static auto ADC_comms = adc::ADC(ADC_intf1, ADC_intf2);
 
-static auto psd = presence_sensing_driver::PresenceSensingDriver{ADC_comms};
+auto at = ot3_tool_list::AttachedTool{};
```

Comment:
[nit] this is passed by-value to the `PresenceSensingDriver` so why not do it inline: `static auto psd = presence_sensing_driver::PresenceSensingDriver{ADC_comms, ot3_tool_list::AttachedTool{}};`